examples/dataflow-data-generator/data-generator-pipeline/data_generator_pipeline.py

```diff
@@ -125,6 +125,19 @@ def run(argv=None):
                 )
         )
 
+    if data_args.write_to_parquet:
+        with open(data_args.schema_file, 'r') as infile:
+            str_schema = json.load(infile)
+        pa_schema = get_pyarrow_translated_schema(str_schema)
+        (rows
```

Comment:
change request in original PR: https://github.com/GoogleCloudPlatform/professional-services/pull/199#discussion_r262758773